docker/README.md

```diff
@@ -1,35 +1,47 @@
 # Docker
 
-This dockerfile runs using dynomite and elasticsearch. The conductor is split into the backend (server) and the frontend (ui). If an image with both of these items combined is desired, build the Dockerfile in the folder serverAndUI
+This dockerfile runs using dynomite and elasticsearch. Conductor is split into backend (server) and frontend (ui) services. The server service is responsible for providing a RESTful interface for all workflow configuration and processing.  The ui service implements a user interface to ease workflow management.
 
-## Building the image
-Building the images:
- - `docker build -t conductor:server ./server`
- - `docker build -t conductor:ui ./ui`
+## Building Conductor
 
-or using compose:
- - `docker-compose build`
+The build process for Java requires 3 docker commands. The first command installs the server source code onto to the Java JDK image.  The second command executes the build process and writes the JAR files back to the project folder at server/build.  Finally, the third command takes the built JAR files and installs them onto the Java 8 JRE Alpine image.  The end result should be a runtime image which is roughly 150 MB in size.
 
-This builds the images:
- - conductor:server - the conductor server and API.
- - conductor:ui - the conductor frontend
+**server:**
 
-## Running conductor
-Running the images:
- - `docker run -p 8080:8080 -d -t conductor:server`
- - `docker run -p 5000:5000 -d -t conductor:ui` (requires elasticsearch running locally)
+`docker build --no-cache -t conductor:server-build -f docker/server/Dockerfile.build .`
 
-Using compose:
-`docker-compose up`
+`docker run --rm -v $(pwd)/server/build:/conductor/server/build conductor:server-build`
 
-## Exiting Compose
-`ctrl+c` will exit docker compose.
+`docker build --no-cache -t conductor:server -f docker/server/Dockerfile .`
 
+**ui:**
 
-To ensure images are stopped do:
- - `docker-compose down`
+`docker build --no-cache -t conductor:ui -f docker/ui/Dockerfile .`
 
-## Running in Interactive Mode
-In interactive mode the default startup script for the container do not run
- - `docker run -p 8080:8080 -t -i conductor:server -`
- - `docker run -p 5000:5000 -t -i conductor:ui -`
+## Running Conductor
+
```

Comment:
I'm thinking that the appropriate way to run Conductor locally is to spin up the entire product service stack using compose.  As users become more savvy with Docker, they can always mimic the behavior of compose with individual commands, but compose is still an easier starting point - especially since you provided the compose file.